lib/Controller/SignallingController.php

```diff
@@ -76,6 +82,11 @@ public function __construct($appName,
 	 * @return JSONResponse
 	 */
 	public function signalling($messages) {
+		$signaling = $this->config->getSignalingServer();
+		if (!empty($signaling)) {
+			throw new \Exception('Internal signaling disabled.');
```

Comment:
Should `return new JSONResponse('Internal signaling disabled.', Http::STATUS_BAD_REQUEST);` instead of throwing?

Reply:
Changed